index.js

```diff
@@ -94,6 +78,38 @@ function init(args) {
   return log(logger);
 }
 
+/**
+ * Initialize logger plugins.
+ * Accepts a comma-delimited list of logging plugins to load and
+ * composes them together to return a single function applying all plugins.
+ *
+ * @return {function}
+ */
+function initPlugins() {
+  const CLAY_LOG_PLUGINS = process.env.CLAY_LOG_PLUGINS || '';
+  const modules = CLAY_LOG_PLUGINS
+    .split(',')
```

Comment:
maybe do the `.trim()` here? in case of `mod1,  _mod2`

Reply:
Yeah good idea.